### PR TITLE
Add explanation of errors and failure for debugging

### LINKED ISSUE #696: Add documentation for tools to return more debugging information
### This is a ...

improvement - something could be better

### This relates to ...

- [X] the **FedRAMP OSCAL Validations**

### User Story

As a developer, engineer, or content owner that must create or maintain FedRAMP data in OSCAL, I want to know information about how to get more warning and error information in stack traces from the `oscal-cli`.

### Goals

- [x] Explain how to opt-in for the `oscal-cli` tool to get debugging information when complex processing fails
- [x] Give commonplace examples in an appropriate place (discuss with team: perhaps wiki, not directly in a README) with common stack trace errors, identify how they are common, and workarounds or resolutions to resolve the error.

### Dependencies

N/A

### Acceptance Criteria

- [x] All FedRAMP Documents Related to OSCAL Adoption (https://github.com/GSA/fedramp-automation) affected by the changes in this issue have been updated.
- [x] A Pull Request (PR) is submitted that fully addresses the goals of this User Story. This issue is referenced in the PR.

### Other information

_No response_

### PR DESCRIPTION
**Pull request by aj-stein-gsa**
# Committer Notes

Closes #696.

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/GSA/fedramp-automation/blob/master/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/GSA/fedramp-automation/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] If applicable, does this PR reference the issue it addresses and explain how it addresses the issue?
- ~If applicable, have all [FedRAMP Documents Related to OSCAL Adoption](https://github.com/GSA/fedramp-automation) affected by the changes in this issue have been updated?~

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### CODE REVIEW

**Comment by wandmagic** [comment]
looks good to me

File: src/validations/constraints/README.md [hunk 1]
```diff
@@ -252,6 +252,145 @@ After validating your FedRAMP OSCAL file, to fix validation errors
...
+```
+
+You should see output like the example below, you can copy-paste this into the other details box of the issue template.
+
+```sh
+REPOSITORY                                       TAG         IMAGE ID      CREATED     SIZE
+ghcr.io/gsa/fedramp-automation/validation-tools  latest      d6f1a0a22474  3 days ago  561 MB
+```
+
+### Manual instal version information
```
Comment (Rene2mt) [inline src/validations/constraints/README.md:358]:
```suggestion
### Manual install version information
```

**Review by Rene2mt — DISMISSED** [review]
One minor typo but not a blocking change as we need this for Wednesday.

**Review by Gabeblis — COMMENTED** [review]
solid

**Pull request merged**